.github/workflows/common.yml

```diff
@@ -3,7 +3,11 @@ on:
   push:
     branches:
       - master
+    paths:
+      - 'python/**'
```

Comment:
Consider adding `'tests/**'` to the path filters to ensure tests are run when test files are modified, even if they're outside the python/ directory.